pyEpiabm/pyEpiabm/tests/test_sweep/test_spatial_sweep.py

```diff
@@ -45,77 +43,96 @@ def test__call__(self, mock_inf, mock_force, mock_poisson, mock_dist,
         mock_inf.return_value = 10.0
         mock_force.return_value = 100.0
         time = 1
-        Parameters.instance().infection_radius = 5000
+        Parameters.instance().infection_radius = 1000
 
+        test_pop = self.pop
         test_sweep = SpatialSweep()
 
         # Assert a population with one cell doesn't do anything
         Parameters.instance().do_CovidSim = False
-        test_sweep.bind_population(self.pop)
+        test_sweep.bind_population(test_pop)
         test_sweep(time)
         self.assertTrue(self.cell_inf.person_queue.empty())
 
         # Add in another cell with a susceptible, but still
         # no infectors so no infection events.
-        self.pop.add_cells(1)
-        self.cell_susc = self.pop.cells[1]
-        self.cell_susc.add_microcells(1)
-        self.microcell_susc = self.cell_susc.microcells[0]
-        self.microcell_susc.add_people(1)
-        self.infectee = self.microcell_susc.persons[0]
-        mock_infectee.return_value = [self.infectee]
-
-        test_sweep.bind_population(self.pop)
+        test_pop.add_cells(1)
+        cell_susc = test_pop.cells[1]
+        cell_susc.add_microcells(1)
+        microcell_susc = cell_susc.microcells[0]
+        microcell_susc.add_people(1)
+        infectee = microcell_susc.persons[0]
+        mock_infectee.return_value = [infectee]
+        mock_choices.return_value = [cell_susc]
+
+        test_sweep.bind_population(test_pop)
         test_sweep(time)
-        self.assertTrue(self.cell_susc.person_queue.empty())
+        self.assertTrue(cell_susc.person_queue.empty())
 
         # Change infector's status to infected
         self.infector.update_status(InfectionStatus.InfectMild)
-        test_sweep.bind_population(self.pop)
+        test_sweep.bind_population(test_pop)
```

Comment:
I am interested, do we _need_ to recall this method each time - surely the population is already bound and we modify the population inplace? The tests seem to work fine without it so I was confused why you've retyped it each time (there may well be a good reason so do let me know if so!)